api/client/utils.go

```diff
@@ -343,3 +344,19 @@ func readBody(stream io.ReadCloser, statusCode int, err error) ([]byte, int, err
 	}
 	return body, statusCode, nil
 }
+
+// getTimestamp tries to parse given string as RFC3339 time
+// or Unix timestamp, if successful returns a Unix timestamp
+// as string otherwise returns value back.
+func getTimestamp(value string) string {
+	format := timeutils.RFC3339NanoFixed
+	loc := time.FixedZone(time.Now().Zone())
+	if len(value) < len(format) {
+		format = format[:len(value)]
+	}
+	if t, err := time.ParseInLocation(format, value, loc); err == nil {
```

Comment:
@runcom makes sense, I'll change. I extracted existing code into a method here. That's why I didn't touch it.